### PR TITLE
All EOE campaign cards (for player decks)

### PR DESCRIPTION
**Pull request by zzorba**
Being released in Europe on Nov 12, this contains all the cards that can go in player decks / or generally have player card backs.

### CODE REVIEW

File: pack/eoe/eoec.json [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+        "pack_code": "eoec",
+        "position": 230,
+        "quantity": 1,
+        "skill_combat": 1,
+        "skill_wild": 1,
+        "slot": "Body",
+        "text": "Fast.\n[reaction] When you reveal a [frost] token during a skill test you are performing, deal 1 damage to Claypool's Furs: Cancel that chaos token and return it to the bag. Reveal a new chaos token.",
+        "traits": "Item. Clothing.",
+        "type_code": "asset",
+        "xp": null
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:705]: Same thing about xp from here to the end.

File: pack/eoe/eoec.json [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+        "faction_code": "neutral",
+        "illustrator": "Mauro Dal Bo",
+        "name": "Cookie's Custom .32",
+        "pack_code": "eoec",
+        "position": 232,
+        "quantity": 1,
+        "skill_combat": 1,
+        "skill_wild": 1,
+        "slot": "Hand",
+        "text": "Fast. Uses (2 ammo).\n[action] Spend 1 ammo: <b>Fight.</b> Either fight with a bas e[combat] skill of 5, or get +2 [combat] for this attack. This attack deals +1 damage.",
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:742]: 'bas e' to 'base' with space after the e before combat.

File: pack/eoe/eoec.json [hunk 3]
```diff
@@ -0,0 +1,869 @@
...
+        "faction_code": "neutral",
+        "flavor": "Professor Dyer kept many sketches depicting the creatures he encountered.",
+        "illustrator": "Germán Nóbile",
+        "name": "Dyer's Sketches",
+        "pack_code": "eoec",
+        "position": 233,
+        "quantity": 1,
+        "skill_intellect": 1,
+        "skill_wild": 1,
+        "text": "Fast. Play during any [fast] window.]\nDraw 3 cards.",
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:762]: Delete spare ] after window.

File: pack/eoe/eoec.json [hunk 4]
```diff
@@ -0,0 +1,869 @@
...
+        "xp": null
+    },
+    {
+        "code": "08737",
+        "cost": 0,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 8,
+        "faction_code": "neutral",
+        "flavor": "Takada left these spare supplies behind. Maybe you can make use of them.",
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:832]: NB here you didn't put a \n between the lines. I think this is the right formatting.

File: pack/eoe/eoec.json [hunk 5]
```diff
@@ -0,0 +1,869 @@
...
+        "illustrator": "Vincent Dutrait",
+        "name": "Takada's Cache",
+        "pack_code": "eoec",
+        "position": 237,
+        "quantity": 1,
+        "skill_wild": 1,
+        "text": "Fast. Play during any [fast] window.]\nGain 3 resources and draw 1 card. Remove Takada's Cache from the game.",
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:839]: Delete extra ] after window.

File: pack/eoe/eoec.json [hunk 6]
```diff
@@ -0,0 +1,869 @@
...
+        "xp": null
+    },
+    {
+        "code": "08738",
+        "cost": 3,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 9,
+        "faction_code": "neutral",
+        "flavor": "\"Nobody knows these machines better than I do.\"",
```
Comment (drawntotheflame) [inline pack/eoe/eoec.json:851]: Delete this flavor (I think it's Takada's, but funny to think Anyu said this)

**Review by drawntotheflame — COMMENTED** [review]
great job @zzorba . Please note the oddity of lots of cards with "xp: null" where we'd normally just remove that line entirely. Probably worth trimming it out, as well as the comma on the preceding line.

File: pack/eoe/eoec.json [hunk 7]
```diff
@@ -0,0 +1,869 @@
...
+        "type_code": "treachery"
+    },
+    {
+        "code": "08714",
+        "cost": null,
+        "deck_limit": 0,
+        "encounter_code": "expedition_team",
+        "encounter_position": 1,
+        "faction_code": "neutral",
+        "flavor": "\"Doubt of the real facts, as I must reveal them, is inevitable.\"\n- H.P. Lovecraft, <i>At the Mountains of Madness</i>",
```
[the remark this answers is not in the document]
Comment (zzorba) [reply pack/eoe/eoec.json:374]: Sounds good, will update everywhere

File: pack/eoe/eoec.json [hunk 8]
```diff
@@ -0,0 +1,869 @@
...
+        "xp": null
+    },
+    {
+        "code": "08717",
+        "cost": null,
+        "deck_limit": 0,
+        "encounter_code": "expedition_team",
+        "encounter_position": 4,
+        "faction_code": "neutral",
+        "flavor": "\"Want to survive the trip? Good.\nThen don't get on my bad side.\"",
```
[the remark this answers is not in the document]
Comment (zzorba) [reply pack/eoe/eoec.json:441]: Thanks, couldn't quite remember what we did with those formatting breaks.

File: pack/eoe/eoec.json (hunk 6 above)
Comment (zzorba) [reply pack/eoe/eoec.json:851]: lol, clever girl

**Comment by zzorba** [comment]
Okay, addressed the feedback.

**Comment by zzorba** [comment]
@Kamalisk I think this one can be merged as well

**Pull request merged**